docs/reference/certificates.rst

```diff
@@ -85,6 +85,16 @@ months and renewal will be attempted within 1 month of expiration.
 
 The *duration* and *renewBefore* parameters must be given in the golang `parseDuration string format <https://golang.org/pkg/time/#ParseDuration>`__.
 
+Some Issuers may set the ``notBefore`` field on their issued X.509 certificate
+to be some time before the time of issuance to fix clock-skew issues, leading
+to the working duration of a certificate to be less than the full duration of
+the certificate. For example, Let's Encrypt sets it to be one hour before the
+time of issuance, so the actual *working duration* of the certificate is 89
+days, 23 hours (the *full duration* remains 90 days). Take care when setting
+the ``renewBefore`` field to be very close to the ``duration`` as this can lead
+to a renewal loop, since the Certificate resource will always be in the renewal
+period.
+
```

Comment:
Sorry to be a pain - can you possibly move this into the "Issuing Certificates" doc under `tasks/` instead of in `reference/`? I think the majority of info in `reference/` will eventually be moved into a new home. There's a few snippets of information that are currently still split across `tasks/` and `reference/` 🙈

Reply:
Sure, will do.